### ISSUE TITLE
[Task] 'Daily Plan' | Member should see which tasks are planned

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#2675]
To distinguish which tasks are planned and which are not. 

Should be visible on the task card as a badge 'Planned' on the next pages:
- Member Profile, tab 'Worked' (should see own tasks with 'Planned + date' badge)
- Member Profile, tab 'Assigned'  (should see own tasks with 'Planned + date' badge)
- Member Profile, tab 'Unassigned'  (should see only 'Planned' badge [without date] and it is planned by anyone)

**Note**: in 'Daily Plan' it does not make any sense to display the badge.

It should display for a user a badge 'Planned + date' ONLY for user's own Profile or Specific person
Only on tabs 'Worked', 'Assigned'

Badge only with word 'Planned' should be displayed on tab 'Unassigned'  (no matter if planned by a few members)

**Important**: planned tasks that are exciting (all past) should be displayed in bright RED color

Managers should see own tasks planned the same color as all team

<img width="985" alt="Screenshot at Jul 01 18-05-27" src="https://github.com/ever-co/ever-teams/assets/107203576/8b58471c-8926-4cb3-9479-7cda07152981">

**Comment by GloireMutaliko21** [comment]
@KostyaEver We already have this badge displayed on task's cards. We could need may to disable it on some pages as describe. 
Recently, I modified it a bit because of some designs broke.
What happened ?
- We displayed it first with the word "Planned" and the added the day after (Today, Tomorrow, or some specific data)
- If the task is planned on 2 or more days, we only display "Planned".
- So some time, especially for other languages (and English too), there was an uncomfortable overflow flow with that "Planned" prefix.

For now, I make it to display only the calendar and the day on that the task is planned. E.g : Today, Tomorrow, 28.07.2024 or Only "Planned" in case of more than one day

CC: @evereq
